config.ru

```diff
@@ -5,7 +5,7 @@ require 'active_record'
 require 'sqlite3'
 require 'tilt/haml'
 require 'uri'
-require 'HTTParty'
+require 'httparty'
```

Comment:
@mvemjsun This is the most "conflicting" change of this PR :-)

Reply:
Glad you find it useful, I have not maintained it for years now but I am sure it could be still useful for many use cases !
Please feel free to improve, I will have a look when I find some time.

Reply:
I found it very useful, because it's the perfect combination of "features"  vs.  "simplicity".

My plan is to use it in QA/STG environments (but not production, because we don't want to mock production)

But I need to mock QA/STG environments, so I will probably create a few more PRs for this.

Now I am working on adding support for a PostgreSQL docker container.

Reply:
Also @mvemjsun I would like to add some support for `docker-compose` .

I tested the `Dockerfile` and the service running from a `docker` container and works perfectly. The problem is that it's _ephemeral_ . So I want to upload also a `docker-compose.yml` file so I can run this with a PostgreSQL container.

I haven't had enough time to finish this, but I will be working on this tomorrow :-)